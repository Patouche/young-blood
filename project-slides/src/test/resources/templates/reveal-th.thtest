%TEMPLATE_MODE HTML5
# ------------ separator comment -----------
%EXACT_MATCH false
# ------------------------------------------
%CONTEXT
page11 = new fr.patouche.slides.SlidePage(0, 'content-1.1')
page12 = new fr.patouche.slides.SlidePage(1, 'content-1.2')
section1 = new fr.patouche.slides.SlideSection(0, 'filename-1', '/path/filename-1', 'content-section-1', { page11, page12 })
page21 = new fr.patouche.slides.SlidePage(0, 'content-2.1')
page22 = new fr.patouche.slides.SlidePage(1, 'content-2.2')
section2 = new fr.patouche.slides.SlideSection(1, 'filename-2', '/path/filename-2', 'content-section-2', { page21, page22 })
sections = { section1, section2 }
# ------------------------------------------
%MESSAGES
presentation.title = Test title
presentation.description = Test description
presentation.author = Test author
demo.title = Test page title
# ------------------------------------------
%INPUT (file:src/main/webapp/WEB-INF/thymeleaf/reveal-th.html)
# ------------------------------------------
%OUTPUT (classpath:templates/reveal-th.expected.html)

