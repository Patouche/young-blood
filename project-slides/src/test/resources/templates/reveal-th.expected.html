<!doctype html>
<html lang="en">
<head>
    <meta charset="utf-8"/>
    <title>Test title</title>
    <meta name="description" content="Test description"/>
    <meta name="author" content="Test author" />
    <meta name="apple-mobile-web-app-capable" content="yes"/>
    <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent"/>
    <link rel="stylesheet" href="/testing/reveal.js/3.1.0/css/reveal.css"/>
    <link rel="stylesheet" href="/testing/reveal.js/3.1.0/css/theme/simple.css" id="theme" />
    <link rel="stylesheet" href="/testing/assets/reveal.css"/>
    <link rel="stylesheet" href="/testing/reveal.js/3.1.0/lib/css/zenburn.css"/>
    <script>
        /*<![CDATA[*/
        if( window.location.search.match( /print-pdf/gi ) ) {
            var link = document.createElement( 'link' );
            link.rel = 'stylesheet';
            link.type = 'text/css';
            link.href = '/testing/reveal.js/3.1.0/css/print/pdf.css';
            document.getElementsByTagName( 'head' )[0].appendChild( link );
        }
        /*]]>*/
    </script>
</head>
<body>
<div class="reveal">
    <!--@thymesVar id="sections" type="java.util.List<fr.patouche.slides.SlideSection>"-->
    <!--@thymesVar id="section" type="fr.patouche.slides.SlideSection"-->
    <div class="slides">
        <section>
            <section>content-1.1</section>
            <section>content-1.2</section>
        </section>
        <section>
            <section>content-2.1</section>
            <section>content-2.2</section>
        </section>
    </div>
</div>
<script src="/testing/reveal.js/3.1.0/lib/js/head.min.js"></script>
<script src="/testing/reveal.js/3.1.0/js/reveal.js"></script>
<script>
    /*<![CDATA[*/
    Reveal.initialize({
        controls: true,
        progress: true,
        history: true,
        center: true,
        theme: Reveal.getQueryHash().theme, // available themes are in /css/theme
        transition: Reveal.getQueryHash().transition || 'default', // default/cube/page/concave/zoom/linear/none
        // Optional libraries used to extend on reveal.js
        dependencies: [
            {
                src: '/testing/reveal.js/3.1.0/lib/js/classList.js',
                condition: function () { return !document.body.classList; }
            },
            {
                src: '/testing/reveal.js-3.2.0/plugin/markdown/showdown.js',
                condition: function () { return !!document.querySelector('[data-markdown]'); }
            },
            {
                src: '/testing/reveal.js/3.1.0/plugin/markdown/markdown.js',
                condition: function () { return !!document.querySelector('[data-markdown]'); }
            },
            {
                src: '/testing/reveal.js/3.1.0/plugin/highlight/highlight.js',
                async: true,
                callback: function () { hljs.initHighlightingOnLoad(); }
            },
            {
                src: '/testing/reveal.js/3.1.0/plugin/zoom-js/zoom.js',
                async: true,
                condition: function () { return !!document.body.classList; }
            },
            {
                src: '/testing/reveal.js/3.1.0/plugin/notes/notes.js',
                async: true,
                condition: function () { return !!document.body.classList; }
            }
        ]
    });
    /*]]>*/
</script>
</body>
</html>